IFC4x3/Properties/w/WeightType_1ACiwCjqf1R8et7_X_P0d0/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="WeightType_1ACiwCjqf1R8et7_X_P0d0" Name="WeightType" UniqueId="4a32ce8c-b74a-416c-8a37-1fe87e6409c0" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel" />
```

Comment:
Penum missing.

Reply:
@fhulin , what information they would like to put here.

Reply:
property has been erased 08487af509e4319afa5d187d589ae5dfd92de558